sdk/eventhub/mock-hub/src/server/mockServer.ts

```diff
@@ -343,7 +347,7 @@ export class MockServer extends EventEmitter {
       : [message];
 
     for (const m of incomingMessages) {
-      if (m.body?.content) {
+      if (m.body.multiple && m.body?.content) {
```

Comment:
Just curious what this is about. Is this the "message was too big for a single transfer frame" or similar condition?

Reply:
This is for batched messages. When I initially wrote this, I noticed `m.body.content` was an array if an AMQP message contained multiple messages. But it turns out sequence bodies also cause `content` to be an array, so without the `multiple` check I was converting every item in `content` to a separate message when the body was a sequence.

Reply:
Add a comment and I think you're good.